doc/python/sphinx/api.rst

```diff
@@ -0,0 +1,153 @@
+API Reference
+=============
+
+.. module:: grpc
+
+Create Client
```

Comment:
Be very, very reticent to use the word "client" when talking about the gRPC Python API. A `grpc.Channel` is not a "client" and a "client" is not a `grpc.Channel`. `grpc.Channel`s are used "on the client side of RPCs", but in the gRPC universe "client" is a mostly-fuzzy, mostly-loosely-defined term (and it should stay that way).